claripy/ast/base.py

```diff
@@ -197,10 +198,11 @@ def _calc_hash(op, args, keywords):
         """
         Calculates the hash of an AST, given the operation, args, and kwargs.
 
-        :param op:          The operation.
-        :param args:        The arguments to the operation.
-        :param keywords:    A dict including the 'symbolic', 'variables', and 'length' items.
-        :returns:           a hash.
+        :param op:                  The operation.
+        :param args:                The arguments to the operation.
+        :param keywords:            A dict including the 'symbolic', 'variables', and 'length' items.
+
```

Comment:
I would prefer not to have a blank line in between the parameters and the return value. rst will treat the list as two separate lists when the newline is inserted.

Reply:
Blank lines removed!
I am going to fix it on angr-docs. The blank line between parameters and the return value seems expected.